### PR TITLE
"over" is ambiguous

### PR DESCRIPTION
**Pull request by koyakei**
For me,  can not identify single means with "over". I imaged, short circuit  it completely or through  it without customize  or with customize.

### CODE REVIEW

File: en/orm/entities.rst [hunk 1]
```diff
@@ -313,7 +313,7 @@ method::
     on.
 
 When using the ``newEntity()`` and ``patchEntity()`` methods in the ``Table``
-objects you also have control over the mass assignment protection. Please refer
+objects you can coustomize mass assignment protection by yourself. Please refer
```
Comment (antograssiot) [inline en/orm/entities.rst]: coustomize -> customize

but I don't see what was "ambiguous" here... I would leave it as is personally.

Comment (koyakei) [inline en/orm/entities.rst]: Make difference  with preposition is difficult to understand .
Set phrase with preposition　is difficult to search (set phrase == look up) in mouse over dictionary.

File: en/orm/entities.rst [hunk 2]
```diff
@@ -313,7 +313,7 @@ method::
     on.
 
 When using the ``newEntity()`` and ``patchEntity()`` methods in the ``Table``
-objects you also have control over the mass assignment protection. Please refer
+objects you can customize mass assignment protection by yourself. Please refer
```
Comment (jadb) [inline en/orm/entities.rst]: Maybe without the `by yourself`?

Comment (koyakei) [inline en/orm/entities.rst]: I can change protection with "Changing Accessible Fields" 
http://book.cakephp.org/3.0/en/orm/saving-data.html#changing-accessible-fields
If changing protection with it  means "without by myself" , your opinion is right..
How do you think?

**Comment by markstory** [comment]
I did an additional tweak in 4ef5ac4ee0c4e07c003529673624f6ae480ccadb

**Pull request merged**